code/API_definitions/device-status.yaml

```diff
@@ -131,32 +153,81 @@ paths:
           $ref: "#/components/responses/Generic500"
         503:
           $ref: "#/components/responses/Generic503"
-  /event-subscriptions:
+  /subscriptions:
     post:
       tags:
-        - Device status event subscription
+        - Device status subscription
       summary: 'Create a device status event subscription for a device'
       description: Create a device status event subscription for a device
-      operationId: createDevicStatusEventSubscription
+      operationId: createDevicStatusSubscription
+      security:
+        - threeLegged:
+          - device-status:roaming-subscriptions:create
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/CreateEventSubscription'
+              $ref: '#/components/schemas/CreateSubscription'
         required: true
+      callbacks:
+        notifications:
+          "{$request.body#/webhook/notificationUrl}":
+            post:
+              tags:
+                - Session notifications callback
+              summary: "Session notifications callback"
+              description: |
+                Important: this endpoint is to be implemented by the API consumer.
+                The Device status server will call this endpoint whenever any device status related event occurs.
+              operationId: postNotification
+              requestBody:
+                required: true
+                content:
+                  application/json:
```

Comment:
We may need to use `application/cloudevents+json` as stated in
https://github.com/cloudevents/spec/blob/main/cloudevents/formats/json-format.md#3-envelope

Reply:
@jlurien Yes make sense.
I will apply.

Do you think we need to go on this detail in the design guideline document as well? I tend to think yes.

Reply:
yes, I think it will be useful to add this and maybe provide a template or artifact as example. I learned about this from some @patrice-conil 's PR indeed